rst2ansi/ansi.py

```diff
@@ -256,8 +256,7 @@ def wrap_current_line(self):
   def depart_paragraph(self, node):
     if self.options.get('wrap_paragraphs', True):
       self.wrap_current_line()
-    if not self.ctx.in_list:
-      self.newline()
+    self.newline()
```

Comment:
This breaks regular lists, for:

```rst
#. numbered
#. list

   #. and a
   #. sublist

#. end

* unordered
* list
```

this always inserts a newline, rendering it as:

```
1. other

2. list

  1. and a

  2. sublist

3. end


• unordered

• list

```

rather than:

```
1. other
2. list
  1. and a
  2. sublist
3. end

• unordered
• list

```